README.md

```diff
@@ -1,2 +1,108 @@
-# doppler-html-editor-api
-API Rest from Doppler HTML Editor
+# Doppler HTML Editor API
+
+## Campaigns
+`GET /accounts/{accountName}/campaigns/{campaignId}/content/design`
+
+Response:
+```
+{
+	"counters":{},
+	"body": {},
+	"schemaVersion": 6
+}
+```
+
+<br>
```

Comment:
Why?

Reply:
Maybe you can use `---` for an `<hr>`? https://www.markdownguide.org/basic-syntax/#horizontal-rule-best-practices

Reply:
Just to add some extra space between

Reply:
It is not a good practice in markdown, you should forget how it is being rendered (it could vary), you should concentrate on the semantic.

Reply:
Fixed!